perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -1474,7 +1474,8 @@
 	 * @param vo virtual organization
 	 * @param namespace used for selecting external system in which guest user account will be created
 	 * @param name a map containing the full name or its parts (mandatory: firstName, lastName; optionally: titleBefore, titleAfter)
-	 * @param password password
+	 * @param password password, if the password is empty, and the `sendActivationLink` is set to true, this method will
```

Comment:
Can you please also update the javadoc for the method above and in above call hierarchy? Thank you.

Reply:
Done.